site/styles/_forms.scss

```diff
@@ -155,11 +155,16 @@ s72-activatedevice-form,
 
 .s72-modal-container {
   .s72-buttons {
+
+    .fa-ellipsis-v {
+      opacity: 0.7;
+    }
+
     .s72-btn {
-      color: $button-text-color;
+      color: var(--body-color);
```

Comment:
You're going to have to add a specific case for the ellipsis because changing this colour and hover colour affects at least one other button:
![image](https://user-images.githubusercontent.com/10665859/165689889-7ac7236b-7397-4af7-a0ad-6d3a15ab0cd0.png)
Design team aren't going to like that lol

Reply:
- Fixed